src/@next/Tag/TagStyle.ts

```diff
@@ -74,10 +99,17 @@ export const TagStyle = styled.div<TagProps>`
 
   @media (max-width: ${Breakpoints.large}) {
     font-size: 12px;
```

Comment:
@michac789 since we are using `subtitle2` for text, I wonder why there is a need for specifying font-size here? Additionally, the font family seems incorrect on storybook

Reply:
removed that (old code), btw font family is incorrect because the pr here: https://github.com/glints-dev/glints-aries/pull/909 has not been merged; there was an issue with testing, me & Nina tried updating but there was a problem, so in the meantime the correct font family are not used in entire glints-aries